packages/contracts-bedrock/snapshots/abi/SuperchainWETH.json

```diff
@@ -408,6 +408,16 @@
     "name": "Withdrawal",
     "type": "event"
   },
+  {
+    "inputs": [],
+    "name": "CallerNotL2ToL2CrossDomainMessenger",
+    "type": "error"
+  },
+  {
+    "inputs": [],
+    "name": "InvalidCrossDomainSender",
+    "type": "error"
+  },
```

Comment:
Hmm the weth contract does not use these errors

idk if there is a better way to omit them, or maybe replace the current ones with these because they are more explicit, wdyt?

Reply:
I'm up to replace them since they're way more explicit